packages/devtools_app/lib/src/shared/common_widgets.dart

```diff
@@ -812,43 +820,21 @@ class AreaPaneHeader extends StatelessWidget implements PreferredSizeWidget {
         alignment: Alignment.centerLeft,
         child: Row(
           children: [
-            DefaultTextStyle(
-              maxLines: maxLines,
-              overflow: TextOverflow.ellipsis,
-              style: theme.textTheme.subtitle2!,
-              child: title,
+            Expanded(
+              child: DefaultTextStyle(
+                maxLines: maxLines,
+                overflow: TextOverflow.ellipsis,
+                style: theme.textTheme.subtitle2!,
+                child: title,
+              ),
             ),
-            ..._buildActions(),
+            AreaPaneHeaderActions(actions),
```

Comment:
Instead of creating another widget for this, just add the actions directly to this Row `...actions`. That gets rid of the nested row that is added by wrapping `actions` with `AreaPaneHeaderActions`